spec/ParseQuery.Aggregate.spec.js

```diff
@@ -845,6 +845,26 @@ describe('Parse.Query Aggregate testing', () => {
       });
   });
 
+  it('match date query - createdAt with bsontype', done => {
```

Comment:
I'm not sure this tests is correct; the title implies "with bsontype", but there is no explicit bsontype being used, it's just a date string?
What's the purpose of `const now` and `const today`, why not just write `new Date().toISOString()`?